src/routers/orderRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const orderController = require("../controllers/orderController");
+const { loginRequired } = require("../utils/auth");
+
+const router = express.Router();
+
+router.get("/:bookId", loginRequired, orderController.checkOrder);
+router.post("", loginRequired, orderController.createOrder);
```

Comment:
엔드포인트 작성할 때 슬래시 작성해주세요!
```suggestion
router.post("/", loginRequired, orderController.createOrder);
```